testplan/web_ui/testing/src/AssertionPane/AssertionTypes/AttachmentAssertions.js

```diff
@@ -16,59 +44,66 @@ import TextAttachment from './TextAttachment.js';
  */
 export const AttachmentAssertion = (props) => {
   const content = getAttachmentContent(props.assertion, props.reportUid);
-  return (
-    <>
-      <Row>
-        <Col lg='6'className={css(styles.contentSpan)}>
-          <span>{content}</span>
-        </Col>
-      </Row>
-    </>
-  );
+  return <ThemeProvider theme={theme}>{content}</ThemeProvider>;
 };
 
 /* Render the attachment content, depending on the filetype. */
 const getAttachmentContent = (assertion, reportUid) => {
-  const fileType = assertion.orig_filename.split('.').pop();
+  const fileType = assertion.orig_filename.split(".").pop();
   const filePath = assertion.dst_path;
+  const description = assertion.description;
   const getPath = getAttachmentUrl(filePath, reportUid);
 
   switch (fileType) {
-    case 'txt':
-    case 'log':
-    case 'out':
-    case 'csv':
+    case "txt":
+    case "log":
+    case "out":
+    case "csv":
       return (
         <TextAttachment
           src={getPath}
           file_name={assertion.orig_filename}
+          file_size={assertion.filesize}
           devMode={reportUid === "_dev"}
         />
       );
 
-    case 'jpeg':
-    case 'jpg':
-    case 'bmp':
-    case 'png':
-        return getImageContent(getPath, assertion.description);
-
+    case "jpeg":
```

Comment:
I'm not sure. Can we check content-type if it is image/*?

Reply:
At this point we do not have the file itself, so to go based on content type we need the backend to set content type. Maybe the result attach can set that from user input. If user do not specify type in the input do semthing similar logic as above just on the backend side. This sounds a good move to me as a second phase, how about open an issuo on this second step?